lib/perl/Genome/Ptero/workflow_tests/parallel-cmd/ptero_workflow_for_process.json

```diff
@@ -85,26 +85,23 @@
                            {
                               "name" : "execute",
                               "parameters" : {
-                                 "commandLine" : [
-                                    "genome",
-                                    "ptero",
-                                    "wrapper",
-                                    "--command-class",
-                                    "Genome::Ptero::TestCommand::ParamAppender",
-                                    "--method",
-                                    "execute",
-                                    "--log-directory",
-                                    "test_logs/parallel-cmd"
-                                 ],
+                                 "command" : "genome ptero wrapper --command-class Genome::Ptero::TestCommand::ParamAppender --method execute --log-directory test_logs/parallel-cmd",
+                                 "cwd" : "/home/archive/dmorton/genome/lib/perl/Genome",
                                  "environment" : {
                                     "FOO" : "bar",
                                     "XGENOME_PTERO_SHELL_COMMAND_SERVICE_URL" : "http://example.com/v1"
                                  },
-                                 "user" : "dmorton",
-                                 "workingDirectory" : "/home/archive/dmorton/genome/lib/perl/Genome"
+                                 "options" : {
+                                    "numProcessors" : "1",
+                                    "resReq" : "select[gtmp>1 && mem>3000] span[hosts=1] rusage[gtmp=1 && mem=3000]"
+                                 },
+                                 "rLimits" : {
+                                    "RSS" : "3000000"
+                                 },
+                                 "user" : "dmorton"
                               },
                               "service" : "job",
-                              "serviceUrl" : "http://example.com/v1"
+                              "serviceUrl" : "http://lsf.apipe-deis.gsc.wustl.edu/v1"
```

Comment:
It's a little strange to me that some of the domains in these `ptero_workflow_for_process.json` files are `example.com` and some are `wustl.edu`.